python/lsst/daf/butler/core/run.py

```diff
@@ -21,8 +21,25 @@
 
 from .utils import slotValuesAreEqual, slotValuesToHash
 
+__all__ = ("Run", )
+
 
 class Run(object):
+    """Represent a processing run.
+
+    Parameters
+    ----------
+    runId : ??
+        ID to associate with this run.
+    registryId : ??
+        ID associated with this `Registry`.
+    collection : ??
+        Collection to use for this run.
+    environment : ??
```

Comment:
string, I guess. Although later we may want to use actual `Dataset` objects for this.